homeassistant/components/sense/__init__.py

```diff
@@ -124,7 +123,12 @@ async def _async_update_trend():
     # This can take longer than 60s and we already know
     # sense is online since get_discovered_device_data was
     # successful so we do it later.
-    asyncio.create_task(trends_coordinator.async_request_refresh())
+    entry.async_on_unload(
+        hass.async_create_background_task(
```

Comment:
MyPy doesn't agree here because I feel like we have too strictly typed `entry.async_on_unload`. We are now forcing all callbacks to return `None`, however, the truth is that we do not care about the return value.

In TypeScript there is an `unknown` type. But I think that in this case we could also consider `Any`. 

Because working around it means that we cannot even use a lambda – as `cancel` will still return a boolean. And PyLint will correctly complain about useless lambda.

The workaround for silly typing is to write it all out.

```python
task = hass.async_create_background_task(...)

def cancel_task()
    task.cancel()

entry.async_on_unload(cancel_task)
```